tests/unit/mesh/mesh_test_utils.hpp

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) 2010-2023, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_MESH_TEST_UTILS
+#define MFEM_MESH_TEST_UTILS
+
+#include "mfem.hpp"
+#include "unit_tests.hpp"
+
+#include <array>
+#include <functional>
+#include <memory>
+
+namespace mfem
+{
+
+/**
+ * @brief Helper function for performing an H1 Poisson solve on a serial mesh,
+ * with homogeneous essential boundary conditions. Optionally can disable a
+ * boundary.
+ *
+ * @param mesh The SERIAL mesh to perform the Poisson solve on
+ * @param order The polynomial order of the basis
+ * @param disabled_boundary_attribute Optional boundary attribute to NOT apply
+ * homogeneous Dirichlet boundary condition on. Default of -1 means no boundary
+ * is disabled.
+ * @return int The number of DOF that are fixed by the essential boundary
+ * condition.
+ */
+int CheckPoisson(Mesh &mesh, int order, int disabled_boundary_attribute = -1);
+
+/**
+ * @brief Helper for counting the number of essential degrees of freedom within
+ * a mesh.
+ *
+ * @tparam FECollection FiniteElementCollection to define the space.
+ * @tparam TDOF Whether or not to counter true degrees of freedom (TDOF) or
```

Comment:
... to count ...

Reply:
Fixed in https://github.com/mfem/mfem/pull/3746/commits/4fe30263e4beb521564a5a2561501e8bbd67900c